bootstrap/090_post_setup.sql

```diff
@@ -379,6 +379,28 @@ CREATE OR REPLACE TASK TASKS.WAREHOUSE_SCHEDULING
     as
     call INTERNAL.UPDATE_WAREHOUSE_SCHEDULES(NULL, NULL);
 
+CREATE OR REPLACE TASK TASKS.SUNDECK_DEBUG
```

Comment:
Let's add a comment to why this exists so that others looking at the code get clarity that this isn't some weird backdoor.